### PR TITLE
Feature/map 222/add/instruction/how/to/create/location

### PR DESCRIPTION
**Pull request by naberalgamma**

### CODE REVIEW

File: frontend/src/components/design/StyledHowToStart.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height: 100%;
+  width: 100%;
+`;
+export const Text = styled(Box)`
+  margin: 20px 0 0 0;
+`;
+export const ButtonTitle = styled(Button)`
+  text-transform: none;
+  font-size: 16px;
+`;
```
Comment (romawht22) [inline frontend/src/components/design/StyledHowToStart.tsx]: Screenshot 2022-07-12 at 12-55-55 MapUa App
Add font-weigth:400 so instruction button will have the same style as other links
3rd
Screenshot 2022-07-12 at 12-58-42 MapUa App

**Review by romawht22 — CHANGES_REQUESTED** [review]

File: frontend/src/components/design/StyledHowToStart.tsx (hunk 1 above)
Comment (naberalgamma) [reply frontend/src/components/design/StyledHowToStart.tsx]: added

**Review by romawht22 — APPROVED** [review]

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
There is an error in the console when clicking on the instruction button. And what do you think about saving only a single icon in the top right corner?
HowAddLocation

**Comment by naberalgamma** [comment]
> There is an error in the console when clicking on the instruction button. And what do you think about saving only a single icon in the top right corner? HowAddLocation

warning fixed, about info icon we will discouse it tomorrow
image

**Review by mazursasha1990 — APPROVED** [review]

**Pull request merged**